src/pages/home/Home.jsx

```diff
@@ -0,0 +1,78 @@
+import React, {useEffect, useState} from 'react';
+import './Home.css';
+import axios from 'axios';
+import {Link} from "react-router-dom";
+import Header from "../../components/header/Header";
+import logo from '../../assets/logo.png';
+
+
+function Home() {
+    const [redditPosts, setRedditPosts] = useState([]);
+    const [catchError, setCatchError] = useState(null);
```

Comment:
Als dit uiteindelijk een boolean wordt, dan zou ik de default waarde aanpassen naar `false`
```suggestion
    const [catchError, setCatchError] = useState(false);
```